tsconfig.json

```diff
@@ -0,0 +1,32 @@
+{
+  "compilerOptions": {
+    // be explicit about the root directory of the project
+    "rootDir": "./src",
+    // Target latest version of ECMAScript.
+    "target": "esnext",
+    // Search under node_modules for non-relative imports.
+    "moduleResolution": "node",
+    // Process & infer types from .js files.
+    "allowJs": true,
+    // Enable strictest settings like strictNullChecks & noImplicitAny.
+    "strict": true,
+    // Import non-ES modules as default imports.
+    "esModuleInterop": true,
+    "noImplicitAny": false,
+    // Ensure that .d.ts files are created by tsc, but not .js files
+    "declaration": true,
+    "emitDeclarationOnly": true,
+    // Ensure that Babel can safely transpile files in the TypeScript project
+    "isolatedModules": true,
```

Comment:
Confirmed that as [per the instructions](https://www.staging-typescript.org/docs/handbook/tutorials/babel-with-typescript.html#babel-for-transpiling-tsc-for-types) being followed, these 3 settings are set to `true` 👍🏻